src/host/ut_host/VtIoTests.cpp

```diff
@@ -362,7 +362,7 @@ void VtIoTests::RendererDtorAndThread()
         // EnablePainting gets called, and if that happens, then the thread will
         // never get destructed. This will only ever happen in the vstest test runner,
         // which is what CI uses.
-        /*Sleep(500);*/
+        Sleep(500);

```

Comment:
Every time I come to look at this pull request you're changing something other than NULLs. We can't review it when we can't be sure that the changes aren't unexpected or undescribed in the PR body.

Reply:
If you're only using one git branch, it might make it difficult to have different kinds of work going on at the same time :smile: